hack/e2e.go

```diff
@@ -83,17 +79,11 @@ func main() {
 	}
 
 	if *build {
-		// The build-release script needs stdin to ask the user whether
```

Comment:
It used to be calling some subscript, but now it's not, so it already has stdin if it needs it.